tests/test_main.cpp

```diff
@@ -369,6 +369,13 @@ struct GodotTestCaseListener : public doctest::IReporter {
 		if (EditorSettings::get_singleton()) {
 			EditorSettings::destroy();
 		}
+		if (EditorPaths::get_singleton()) {
+			EditorPaths::free();
+		}
+		if (EditorSettings::get_singleton()) {
+			EditorSettings::destroy();
+		}
```

Comment:
This duplicates what's already done above. Does `destroy` fail the first time? That sounds like a bug.

Reply:
You're right, that was a copy paste mistake. Fixed now.